test/stage-options.test.ts

```diff
@@ -197,3 +197,64 @@ describe('job settings', () => {
     });
   });
 });
+
+describe('role settings', () => {
+  test('can specify gha role', () => {
```

Comment:
this test isn't actually testing any features implemented in this PR, is it?

Reply:
No. Sorry. This was meant to be a snapshot test to confirm the GHA pipeline role. I'll improve the tests